pkg/apply/event/event.go

```diff
@@ -129,9 +132,11 @@ const (
 )
 
 type PruneEvent struct {
-	Type      PruneEventType
-	Operation PruneEventOperation
-	Object    runtime.Object
+	Type       PruneEventType
+	Operation  PruneEventOperation
+	Object     runtime.Object
+	ObjectMeta object.ObjMetadata
```

Comment:
We usually name this `Identifier`. Not sure if it is a great name, but we should probably just be consistent.

Reply:
Ack, will update it.